src/test_data/get_users/user.csv

```diff
@@ -0,0 +1,7 @@
+id;email;forename;surname;is_admin;is_blender;salt;password_hash;archived_at;deleted_at;created_at;updated_at;
+2;test@email.fi;Tester;Blender;0;1;salt;hash!"#€;;;2020-01-19 03:14:07;2020-01-19 03:14:07;
+3;testi2@email.fi;Teemu;Testaaja;0;1;salt;hash!"#€;;;2020-01-19 03:14:07;2020-01-19 03:14:07;
+deletedUser;alreadyin@use.fi;Pro;Blendaaja;0;1;salt;hash!"#€;;2020-01-19 05:15:07;2020-01-19 03:15:07;2020-02-19 03:17:07;
+archivedUser;alreadynotin@use.fi;Pro;Blendaaja;0;1;salt;hash!"#€;2020-01-19 07:15:07;;2020-01-19 03:15:07;2020-02-19 03:17:07;
```

Comment:
Täällä taitaa olla sama sisältö kuin tuolla get_user-hakemiston puolella? Eikö ne testikeissit voisi viitata yhteiseen testidataan?

Reply:
On sama sisältö ja voisi viitata mutta mielestäni parempi silti pitää endpointtien testidata erikseen. Tällöin ei varmastikaan testidatan muuntelu vaikuta muihin testeihin.

Reply:
Tulevaisuudessa kannat voi erota. Parempi pitää erillään

Reply:
Noi endpointit on tehty saman datan hakemiseen kahdella eri tavalla. Keksikää esimerkki miten noi kannat tulee eroamaan niin hyväksyn.

Reply:
Toisaalta myös periaatteen vuoksi voidaan haluta pitää ne erillään.

Reply:
> Noi endpointit on tehty saman datan hakemiseen kahdella eri tavalla. Keksikää esimerkki miten noi kannat tulee eroamaan niin hyväksyn.

Esim. updateUser testit ei vaadi sitä, että testidatassa on poistettu käyttäjä kun taas getUser ja getUsers vaatii (mistä tuli mieleen että pitäisi varmaan testata että updateUser ei voi muuttaa poistetun käyttäjän dataa)